justspacesapp/views.py

```diff
@@ -26,13 +26,7 @@ class StudyCreate(CreateView):
     success_url = '/'
 
 
-class SurveyListEdit(ListView):
+class SurveyList(ListView):
```

Comment:
This looks great! I'll open an issue to remind us to account for query params, i.e., to filter between published and unpublished (or whatever adjective we use).

Reply:
Thank you!